apps/bedrock/k8s/frankfurt/bedrock-prod-svc.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: v1
+items:
+- apiVersion: v1
+  kind: Service
+  metadata:
+    annotations:
+      kubectl.kubernetes.io/last-applied-configuration: |
+        {"apiVersion":"v1","kind":"Service","metadata":{"annotations":{},"labels":{"app":"bedrock-prod"},"name":"bedrock-nodeport","namespace":"bedrock-prod"},"spec":{"ports":[{"name":"https","port":443,"targetPort":8000}],"selector":{"app":"bedrock-prod","type":"web"},"type":"NodePort"}}
+    creationTimestamp: 2017-06-16T16:48:43Z
+    labels:
+      app: bedrock-prod
+    name: bedrock-nodeport
+    namespace: bedrock-prod
+    resourceVersion: "690478"
+    selfLink: /api/v1/namespaces/bedrock-prod/services/bedrock-nodeport
+    uid: a8381c22-52b3-11e7-8dc9-02859405480d
+  spec:
+    clusterIP: 100.67.37.244
+    ports:
+    - name: https
+      nodePort: 30588
+      port: 443
+      protocol: TCP
+      targetPort: 8000
+    selector:
+      app: bedrock-prod
+      type: web
+    sessionAffinity: None
+    type: NodePort
+  status:
+    loadBalancer: {}
+- apiVersion: v1
+  kind: Service
+  metadata:
+    annotations:
+      router.deis.io/domains: bedrock-prod,www.mozilla.org
```

Comment:
the deis annotations should eventually be removed, but they're "mostly" harmless for now.